### DISCUSSION TITLE
need help to get sqlite to work, currently get an error saying sqlite driver not found

### DISCUSSION THREAD

**Discussion by nerminba** [leokhoa/laragon#452] [Q&A] [answered]
Hi I have sqlite3 enabled as php extension, php 8.2 is installed

![image](https://user-images.githubusercontent.com/443080/227510282-88ef34b9-fa00-40ce-aead-4198e8e76c26.png)

when I tried to access my local https://myapp.test environment after I clicked on login and submitted the login form I got this screen where I notice that sqlite driver is missing. 
![image](https://user-images.githubusercontent.com/443080/227510544-8f498b3c-73c1-4aed-8a86-3bbc9434a2f0.png)

Prior to running this in Win11 with Laragon, I had this same project setup with wsl2 instance that has ubuntu22 installed and there I did not run into same issue with sqlite. Now that I've moved to project to win11 and using laragon, I'm trying to get past this issue. Greatfull if someone can help me get this solved.

**Comment by nerminba** [comment]
And I've solved it. I'll leave this thread for the future. On windows I had to enable pdo_sqlite extension as well, easily done with Laragon. Thank you :)

**Comment by codename-12** [comment]
hello, can u explain how to do it? tia

**Reply by Pen-y-Fan** [reply, marked as answer]
It's the **pdo_sqlite** extension, higher than **sqlite3**. Laravel uses the PDO version of extensions.

Laragon menu > PHP > Extensions > tick **pdo_sqlite**

**Reply by nerminba** [reply]
thank you, that was quick! :)

**Comment by nizamipunjabpolice** [comment]
I want to use mysql instead of sqlite. When I quick add laravel project it gives me error of could not find driver.. I want to add mysql as default db whenever i create new project. please help me in this regard

**Reply by nerminba** [reply]
Hi, I believe mysql may be default driver for Laravel, so you have to enable pdo_mysql php extension. If you for example use tool like Laragon and you are on Windows, take a look at attached screenshot how to enable this extension via GUI. Hope that helps. Actually enable every extension I have enabled in case you still get some errors, and then it should be good hopefully :)
![image](https://github.com/leokhoa/laragon/assets/443080/1999a5cb-129a-4b5f-ac80-1dc9bf1dd0a7)

**Reply by Pen-y-Fan** [reply]
To use MySQL in Laravel 11 update your .env file:

```ini
DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=your_project_database
DB_USERNAME=root
DB_PASSWORD=
```

Laravel changed the default database to Sqlite, so you need to change it back if needed.
